### PR TITLE
contributing : making changes

### PR DESCRIPTION
**Pull request by nplanel**
Signed-off-by: Nicolas PLANEL <nplanel@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
 
 https://github.com/skydive-project/skydive/issues
 
+Making Changes:
```
Comment (safchain) [inline CONTRIBUTING.md]: I would move this before "How to submit"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
...
+---------------
+
+Make sure that the tests and the linters pass:
```
Comment (safchain) [inline CONTRIBUTING.md]: Before pushing your changes, please make sure.....

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
...
+
+make fmt
```
Comment (safchain) [inline CONTRIBUTING.md]: * make fmt
* make test
* make.functionals

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
...
+make test
+make test.functionals
```
Comment (safchain) [inline CONTRIBUTING.md]: for this last one I'm not sure or we should warn the user that it is going to create local network resources

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
...
+
+Make your changes, then ensure make test still pass. If you're satisfied with your changes, push them to your fork.
```
Comment (safchain) [inline CONTRIBUTING.md]: I would say:

Once ready push your changes to your Skydive fork our CI will check them.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
...
+Our CI platform will check for you to ensure that tests and linters still pass
+
+We're much more likely to approve your changes if you:
+
+* Add tests for new functionality.
```
Comment (safchain) [inline CONTRIBUTING.md]: for a new...

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -40,6 +40,24 @@ Remember that contributions are always welcome!
...
+* Write a good commit message.
```
Comment (safchain) [inline CONTRIBUTING.md]: commit message, please see how to write bellow

File: CONTRIBUTING.md (hunk 6 above)
Comment (nplanel) [inline CONTRIBUTING.md]: for new is correct too

File: CONTRIBUTING.md (hunk 1 above)
Comment (nplanel) [inline CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (nplanel) [inline CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 3 above)
Comment (nplanel) [inline CONTRIBUTING.md]: make functionals didn't exist

File: CONTRIBUTING.md (hunk 4 above)
Comment (nplanel) [inline CONTRIBUTING.md]: done by a please note ...

File: CONTRIBUTING.md (hunk 5 above)
Comment (nplanel) [inline CONTRIBUTING.md]: ... check your pull request

File: CONTRIBUTING.md (hunk 7 above)
Comment (nplanel) [inline CONTRIBUTING.md]: done

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -4,6 +4,27 @@ Contributing to Skydive
 Your contributions are more than welcome. Please read the following notes to
 know about the process.
 
+Making Changes:
+---------------
+
+Before pushing your changes, please make sure the tests and the linters pass:
+
+make fmt
```
Comment (safchain) [inline CONTRIBUTING.md]: add * before each line otherwise it is not readable in GitHub

Comment (nplanel) [inline CONTRIBUTING.md]: done

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -4,6 +4,27 @@ Contributing to Skydive
...
 know about the process.
 
+Making Changes:
+---------------
+
+Before pushing your changes, please make sure the tests and the linters pass:
+
+* make fmt
+* make test
+* make test.functionals
```
Comment (masco-enovance) [inline CONTRIBUTING.md]: i feel it is better to mention that we need to export the SKYDIVE_ANALYZERS before running the tests? if not require ignore this :)

Comment (lebauce) [inline CONTRIBUTING.md]: No need to export SKYDIVE_ANALYZERS when using the 'standalone mode'.

make test.functionals VERBOSE=true TIMEOUT=10m ARGS='-standalone'

Comment (nplanel) [inline CONTRIBUTING.md]: I will add a rule for that
make functional   => make test.functionals VERBOSE=true TIMEOUT=10m ARGS='-standalone'

**Pull request merged**